src/cmd/build.js

```diff
@@ -37,14 +39,56 @@ export type PackageCreatorParams = {
   artifactsDir: string,
 };
 
+export type LocalizedNameParams = {
+  messageFile: string,
+  manifestData: ExtensionManifest,
+}
+
 export type PackageCreatorFn =
     (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
 
+export async function getDefaultLocalizedName(
+  {messageFile, manifestData}: LocalizedNameParams
+): Promise<string> {
+
+  let messageData: any;
```

Comment:
try to only use `any` if you have a really good reason :) The `messageData` should always be a string but some internal Node functions return Buffer instances (which are compatible with strings) so I think it should be `messageData: string|Buffer`